datasets/shield.md

```diff
@@ -1,5 +1,121 @@
-# Work in Progress
+# Analyzing data from SHIELD studies
 
-This document is a work in progress.
-The work is being tracked in
-[this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1358286)
+This article introduces the datasets that are useful for analyzing SHIELD studies.
+After reading this article,
+you should understand how to answer questions about
+study enrollment,
+identify telemetry from clients enrolled in an experiment,
+and locate telemetry from add-on studies.
+
+## Table of contents
+
+<!-- toc -->
+
+## Dashboards
+
+[Firefox Test Tube](https://firefox-test-tube.herokuapp.com/)
+shows live enrollment activity and a selection of browser activity metrics
+for active pref-flip (but not add-on) experiments.
+
+## Experiment slugs
+
+Each experiment is associated with a slug,
```

Comment:
One normandy quirk is that every pref-flip experiment has a slug, while addon studies have a "name". Not sure why this is the case but so it goes.

Reply:
Can I assert that add-ons have a slug specified by a `name` field?